src/tests/ftest/dfuse/bash.py

```diff
@@ -138,8 +139,16 @@ def run_bashcmd(self, il_lib=None, compatible_mode=False, nobypass=False):
             'fio --readwrite=randwrite --name=test --size="2M" --directory '
             f'{fuse_root_dir}/ --bs=1M --numjobs="1" --ioengine=libaio --iodepth=16'
             '--group_reporting --exitall_on_error --continue_on_error=none',
-            f'curl "https://www.google.com" -o {fuse_root_dir}/download.html',
         ]
+        # If set, use the HTTPS_PROXY for curl command
+        https_proxy = os.environ.get('HTTPS_PROXY')
+        if https_proxy:
+            proxy_option = f'--proxy "{https_proxy}"'
+        else:
+            proxy_option = ''
+        cmd = f'curl "https://www.google.com" -o {fuse_root_dir}/download.html {proxy_option}'
+        commands.append(cmd)
```

Comment:
FYI if you find more places need this, there is a more global place where we could export HTTPS_PROXY for all of our clush commands. Just let me know. Otherwise, I don't want to hold this up with more complexity